Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3904,8 +3937,11 @@ def gen2021HiMix(fragment,howMuch):
                         istep = step+preventReuseKeyword
                         if 'FastSim' not in k and 'Run3FS' not in k and s+'INPUT' not in steps and s in baseDataSetReleaseBetter and defaultDataSets[key] != '' and \
```

Comment:
This was modified by #38660 with https://github.com/cms-sw/cmssw/pull/38660/files#diff-b0dce787f15ea8526a7b5585e797ba4b9e17ba829550c8283b5b5f97cba58b04R3877:
```suggestion
                        if 'FastSim' not in k and s+'INPUT' not in steps and s in baseDataSetReleaseBetter and defaultDataSets[key] != '' and \
```
Please either fix or confirm that it is not needed here